SampleApp/SampleApp/TabbedPageView.xaml

```diff
@@ -5,4 +5,5 @@
     xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="SampleApp.TabbedPageView">
     <local:HomeViewController Title="{x:Static resources:AppResources.home_title}" IconImageSource="home"/>
     <local:PushSettingsViewController Title="{x:Static resources:AppResources.settings_title}" IconImageSource="settings"/>
+    <local:MessageCenterViewController Title="{x:Static resources:AppResources.message_center_title}" IconImageSource="inbox"/>
```

Comment:
Can you please put the Message Center tab in the middle of the tab bar, to mach our other sample apps?

Reply:
Ok Done 👍